casper/src/test/scala/coop/rchain/casper/util/rholang/DeployIdTest.scala

```diff
@@ -42,13 +43,7 @@ class DeployIdTest extends FlatSpec with Matchers {
     val d = deploy(sk, s"""new return, deployId(`rho:rchain:deployId`) in { return!(*deployId) }""")
     val result =
       runtimeManager
-        .use(
-          mgr =>
-            for {
-              hash <- RuntimeManager.preGenesisStateHashFixed.pure[Task]
-              res  <- mgr.captureResults(hash, d)
-            } yield res
-        )
+        .use(mgr => mgr.captureResults(emptyRootHash.toByteString, d))
```

Comment:
tests like these don't need the registry so we could just use emptyRootHash for testing.